doc/examples/go_desktop/lib/game.dart

```diff
@@ -6,10 +6,10 @@ import 'package:flutter/material.dart';
 class MyGame extends Game {
   static final Paint paint = Paint()..color = const Color(0xFFFFFFFF);
 
-  var movingLeft = false;
-  var movingRight = false;
-  var movingUp = false;
-  var movingDown = false;
+  bool movingLeft = false;
+  bool movingRight = false;
+  bool movingUp = false;
+  bool movingDown = false;
 
```

Comment:
This is the opposite of `omit_local_variable_types` (but in fields, not local vars). What rule required this change?

Reply:
I commented elsewhere, but basically:
local variables are never explicitly typed. fields/methods are always explicitly typed.